nextflow_schema.json

```
@@ -48,24 +48,34 @@
                 },
                 "min_counts": {
                     "type": "integer",
-                    "description": "Minimum number of variant observations required.",
+                    "description": "minimum counts for variant to be recognized. All variants below min_counts will be set to 0",
                     "minimum": 1,
                     "default": 3
                 },
+		"sliding_window_size": {
+                    "type": "integer",
+                    "description": "To flatten graphs in plots (e.g. GLOBAL_POS_BIASES_COUNTS function)",
```

```suggestion
                    "description": "To flatten graphs in plots (e.g. `GLOBAL_POS_BIASES_COUNTS` function)",
```
you can use markdown syntax for the description